pyproject.toml

```diff
@@ -34,6 +34,13 @@ profile = "black"
 combine_as_imports = true
 ensure_newline_before_comments = false
 
+[tool.pylint.messages_control]
+disable = [
+    "missing-function-docstring",
```

Comment:
The original code has no doc string at all, 
It make sense to me to disable it ATM.